go/vt/vtgate/planbuilder/operators/derived.go

```diff
@@ -218,3 +220,12 @@ func addToIntSlice(columnOffset []int, valToAdd int) ([]int, int) {
 	columnOffset = append(columnOffset, valToAdd)
 	return columnOffset, len(columnOffset) - 1
 }
+
+// TODO: REMOVE
```

Comment:
this should be cleaned up before we merge this PR

Reply:
Let's clean it up :p

Reply:
we are fixing this here: https://github.com/vitessio/vitess/pull/12887